dlp/snippets/inspect/inspect_test.go

```diff
@@ -46,26 +196,93 @@ func TestInspectTextFile(t *testing.T) {
 
 	got := buf.String()
 	if want := "Info type: PHONE_NUMBER"; !strings.Contains(got, want) {
-		t.Errorf("got %q, want %q", got, want)
+		t.Errorf("inspectTextFile got %q, want %q", got, want)
 	}
 	if want := "Info type: EMAIL_ADDRESS"; !strings.Contains(got, want) {
-		t.Errorf("got %q, want %q", got, want)
+		t.Errorf("inspectTextFile got %q, want %q", got, want)
 	}
 }
 
-func TestInspectImageFile(t *testing.T) {
-	tc := testutil.SystemTest(t)
-	buf := new(bytes.Buffer)
-	err := inspectImageFile(buf, tc.ProjectID, "testdata/test.png")
+type Item struct {
+	Description string
+}
+
+const (
+	harmlessTable = "harmless"
+	harmfulTable  = "harmful"
+	bqDatasetID   = "golang_samples_dlp"
+)
+
+func createBigqueryTestFiles(projectID, datasetID string) error {
```

Comment:
might want to take t as parameter, t.Helper()

then t.Log before errors to aid debugging.

Reply:
Done. Just called `t.Fatal` directly, since that's how it was handled by the caller.